README.md

```diff
@@ -1 +1,68 @@
-# Microverse-todo-list-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse To-Do list
+
+> This is a Microverse (@microverseinc) project in which I created a To-do list using webpack.
+
+## Requirements
+
+Build a Todo list using:
+
+- HTML5
+- CSS3
+- JavaScript (ES6)
+- Webpack
+- Linters (Lighthouse, Webhint, Stylelint, Eslint)
+
```

Comment:
- [ ]  Kindly add font-awesome to the stack used to create a project in the readme file

Reply:
Thank you, I forgot about it ! I will add it 😊